### ISSUE TITLE
`npm run typecheck` fails on fresh project

### ISSUE THREAD

**Issue by ElanHasson** [remix-run/blues-stack#190]
### Have you experienced this bug with the latest version of the template?

yes

### Steps to Reproduce

Create a bluestack via  `npx create-remix@latest` and choose bluestack. Follow instructions, run `npm run typecheck`.

### Expected Behavior

Tests should pass out of the box.

### Actual Behavior

Tests fail:

```shell
npm run typecheck 

> typecheck
> tsc && tsc -p cypress

error TS2688: Cannot find type definition file for '@testing-library/cypress'.
  The file is in the program because:
    Entry point of type library '@testing-library/cypress' specified in compilerOptions

  cypress/tsconfig.json:15:34
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is entry point of type library specified here.

error TS2688: Cannot find type definition file for 'cypress'.
  The file is in the program because:
    Entry point of type library 'cypress' specified in compilerOptions

  cypress/tsconfig.json:15:23
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                             ~~~~~~~~~
    File is entry point of type library specified here.


Found 2 errors.
```

**Comment by Tonours** [comment]
Hello 👋 ! I've encountered the same issues. They seem related to TypeScript 5.1.3. Everything functions perfectly when I downgrade to TypeScript 5.0.4.

**Comment by ElanHasson** [comment]
Heh so much for semantic versioning.

Thanks

**Comment by kinggoesgaming** [comment]
> Heh so much for semantic versioning.
> 
> Thanks

Fun fact: TypeScript does not follow SemVer, even though at first glance it seems they do.

Source: https://github.com/microsoft/TypeScript/issues/14116

**Comment by ElanHasson** [comment]
TIL thanks @kinggoesgaming

**Comment by MichaelDeBoey** [comment]
Will be fixed by #194
